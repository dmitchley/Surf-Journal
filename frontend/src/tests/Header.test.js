import { expect, it } from "vitest";
import Header from "../components/Header";

it("Header JSX", () => {
  const result = Header(Header);
  expect(result).toMatchSnapshot();
});
