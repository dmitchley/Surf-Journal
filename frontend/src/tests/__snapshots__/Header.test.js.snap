// Vitest Snapshot v1

exports[`Header JSX 1`] = `
<header
  className="sticky top-0 bg-white border-b border-slate-200 z-30"
>
  <div
    className="px-4 sm:px-6 lg:px-8"
  >
    <div
      className="flex items-center justify-between h-16 -mb-px"
    >
      <div
        className="flex"
      >
        <button
          aria-controls="sidebar"
          className="text-slate-500 hover:text-slate-600 lg:hidden"
          onClick={[Function]}
        >
          <span
            className="sr-only"
          >
            Open sidebar
          </span>
          <svg
            className="w-6 h-6 fill-current"
            viewBox="0 0 24 24"
            xmlns="http://www.w3.org/2000/svg"
          >
            <rect
              height="2"
              width="16"
              x="4"
              y="5"
            />
            <rect
              height="2"
              width="16"
              x="4"
              y="11"
            />
            <rect
              height="2"
              width="16"
              x="4"
              y="17"
            />
          </svg>
        </button>
      </div>
      <div
        className="flex items-center"
      >
        <hr
          className="w-px h-6 bg-slate-200 mx-3"
        />
        <UserMenu />
      </div>
    </div>
  </div>
</header>
`;
